src/patternfly/components/Select/examples/index.js

```diff
@@ -61,6 +64,14 @@ export default (props) => {
       <Example heading="Select - single (expanded)" handlebars={SelectSingleExpandedExampleRaw} minHeight="20em">
         {selectSingleExpandedExample}
       </Example>
+      <Example
+        className="is-expanded-top"
+        heading="Select - single (top expanded)"
+        handlebars={SelectSingleTopExpandedExampleRaw}
+        minHeight="20em"
```

Comment:
You don't need this now that you're using `className="is-expanded-top"`

Reply:
Thanks a lot. I did miss that! :smile: